### DISCUSSION TITLE
French translation

### DISCUSSION THREAD

**Discussion by ihhub** [ihhub/fheroes2#3544] [Q&A]
Hi @LeHerosInconnu,

In the near future we are going to add proper translation support. Could you please give me translated to French common words used in the game? I'll try to develop a prototype of translation to French based on existing set of characters.

**Comment by LeHerosInconnu** [comment]
Hello @ihhub,

> Hi @LeHerosInconnu,
> 
> In the near future we are going to add proper translation support. Could you please give me translated to French common words used in the game? I'll try to develop a prototype of translation to French based on existing set of characters.

Here are some texts copied from the French version (French in game.txt) and the English version (English in game.txt) of the game (The Succession Wars).
I corrected the characters used in the French text (French corrected.txt) with the character correspondence (Character correspondence from french in game to french corrected.txt).

I hope this is suitable for what you want to do.

[French translation 01.zip](https://github.com/ihhub/fheroes2/files/6560896/French.translation.01.zip)
